File: common/src/main/java/com/github/xiaogegechen/common/util/LogUtil.java
package com.github.xiaogegechen.common.util;

import android.util.Log;

import com.github.xiaogegechen.common.base.AppConfig;

/**
 * log工具
 */
public class LogUtil {

    public static void d(String tag, String message){
        if(AppConfig.LOG){
            Log.d(tag, message);
        }
    }

    public static void i(String tag, String message){
        if(AppConfig.LOG){
            Log.i(tag, message);
        }
    }

    public static void w(String tag, String message){
        if(AppConfig.LOG){
            Log.w(tag, message);
        }
    }

    public static void e(String tag, String message){
        if(AppConfig.LOG){
            Log.e(tag, message);
        }
    }
}

File: network/src/main/java/com/github/xiaogegechen/network/LogUtil.kt
package com.github.xiaogegechen.network

import android.util.Log

internal object LogUtil {

    @JvmStatic
    private var sOpenLog = true

    @JvmStatic
    fun openLog(b: Boolean){
        sOpenLog = b
    }

    fun d(tag: String, message: String){
        if(sOpenLog){
            Log.d(tag, message)
        }
    }
}